README.md

```diff
@@ -119,6 +116,9 @@ DeviceManagementManagedDevices.Read.All<br/>
 </b>
 <br/>
 
+To add devices to Groups the service prinicpal must have this permission:
+<b>GroupMembers.ReadWriteAll<br/>
```

Comment:
Should read GroupMembers.ReadWrite.All for the AU or the tenant. OR the service principal has group owner rights.